.bazelrc

```diff
@@ -58,4 +58,4 @@ build:libc++ --define force_libcpp=enabled
 build:libc++ --linkopt -fuse-ld=lld
 
 # Test options
-test --test_env=HEAPCHECK=normal --test_env=PPROF_PATH
+test --test_env=HEAPCHECK=normal --test_env=PPROF_PATH --experimental_remap_main_repo
```

Comment:
nit: since 'test' options inherit from 'build' options (at least according to [.bazelrc doc](https://docs.bazel.build/versions/master/guide.html#bazelrc)), can you leave out specifying the flag here?